internal/venice-common/src/main/java/com/linkedin/venice/ConfigKeys.java

```diff
@@ -183,6 +183,11 @@ private ConfigKeys() {
   public static final String ENABLE_ACTIVE_ACTIVE_REPLICATION_AS_DEFAULT_FOR_HYBRID_STORE =
       "enable.active.active.replication.as.default.for.hybrid.store";
 
+  /**
+   * Store-level config to enable blob transfers (feature flag)
+   */
+  public static final String ENABLE_BLOB_TRANSFER_FOR_BATCH_ONLY_STORE = "enable.blob.transfer.for.batch.only.store";
```

Comment:
Actually one more thought is since this is a store-level property, we need to add it to [StoreMetaValue](https://github.com/linkedin/venice/blob/f07d0c13edcd64e730a88925390b4c739f4ac4f5/internal/venice-common/src/main/resources/avro/StoreMetaValue/v18/StoreMetaValue.avsc) so it also lives on ZK. What do you think?